acceptance-test/user-pathways/upload-file/upload-file.test.js

```diff
@@ -68,117 +72,136 @@ describe('Upload File(s)', () => {
-
-            // pv-under-age-at-time-of-exploitation
-            await page.waitForSelector(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await page.click(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await clickContinueButton(1);
-
-            // what-happened
-            await page.waitForSelector(WHAT_HAPPENED_INPUT);
-            await page.$eval(WHAT_HAPPENED_INPUT, (element) => {
-                element.value = 'Test input regarding details of exploitation';
-            });
-            await clickContinueButton(1);
-
-            // where-exploitation-happened
-            await page.waitForSelector(EXPLOITED_IN_UK_OPTION);
-            await page.click(EXPLOITED_IN_UK_OPTION);
-            await clickContinueButton(1);
-
-            // current-pv-location
-            await page.waitForSelector(CURRENT_PV_LOCATION_UK_REGION);
-            await page.$eval(CURRENT_PV_LOCATION_UK_REGION, (element) => {
-                element.value = 'Rutland';
-            });
-            await clickContinueButton(1);
-
-            // who-exploited-pv
-            await page.waitForSelector(WHO_EXPLOITED_PV);
-            await page.$eval(WHO_EXPLOITED_PV, (element) => {
-                element.value = 'Test details about exploiter(s)';
-            });
-            await clickContinueButton(1);
-
-            // types-of-exploitation
-            await clickContinueButton(1);
-
-            // any-other-pvs
-            await page.waitForSelector(ANY_OTHER_PVS_NO_OPTION);
-            await page.click(ANY_OTHER_PVS_NO_OPTION);
-            await clickContinueButton(1);
-
-            // reported-to-police
-            await page.waitForSelector(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await page.click(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await clickContinueButton(1);
-
-            // Run through the skeleton until we reach the upload page
-            await clickContinueButton(13);
-
-            await page.waitForSelector(UPLOAD_DOCUMENT_PAGE_2_YES_OPTION);
-            await page.click(UPLOAD_DOCUMENT_PAGE_2_YES_OPTION);
-            await page.click(CONTINUE_BUTTON);
-
-            await page.waitForSelector(UPLOAD_DOCUMENT_PAGE_3_UPLOAD_FILE_INPUT);
-            const input = await page.$(UPLOAD_DOCUMENT_PAGE_3_UPLOAD_FILE_INPUT);
-            await input.uploadFile(TEST_FILE_PATH());
-
-            await page.$eval(UPLOAD_DOCUMENT_PAGE_3_UPLOAD_FILE_DESCRIPTION, (element) => {
-                element.value = 'NRM Test File example';
-            });
-            await page.click(UPLOAD_DOCUMENT_PAGE_3_UPLOAD_FILE_BUTTON);
-
-            await page.waitForSelector(UPLOAD_DOCUMENT_PAGE_4_NO_OPTION);
-            await page.click(UPLOAD_DOCUMENT_PAGE_4_NO_OPTION);
+    /**
+     * Run a sequence of actions to simulate the completion of the first half
+     * of the NRM form
+     *
+     * @returns {Promise}
+     */
+    async function completeNrmFormPart1() {
```

Comment:
Should this say `completeNrmFormUpto` and then the name of the page otherwise it's not obvious?